prediction_market_agent/agents/skew_agent/deploy.py

```
@@ -0,0 +1,94 @@
+    FixedInterval,
+    TradeInterval,
+)
+from prediction_market_agent_tooling.gtypes import USD, Probability
+from prediction_market_agent_tooling.loggers import logger
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket, SortBy
+from prediction_market_agent_tooling.markets.data_models import ProbabilisticAnswer
+from prediction_market_agent_tooling.markets.markets import MarketType
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    FilterBy,
+    OmenSubgraphHandler,
+    SortBy,
```

Ruff (0.12.2)

Redefinition of unused `SortBy` from line 16

(F811)